### PR TITLE
Jenkins course added in courses/free-courses-en.md

### PR DESCRIPTION
**Pull request by Manish-Syal123**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo
Add resources
### Description
As the courses are from youtube so it is free and the other one is the site that provides free courses in English
### Why is this valuable (or not)?
Jenkins is used to build and test the product continuously, so developers can continuously integrate changes into the build. Jenkins is the most popular open source CI/CD tool on the market today and is used in support of DevOps, alongside other cloud native tools.
### How do we know it's really free?
As the courses are from youtube so it is free and the other one is the site that provides some free courses in English
### For book lists, is it a book? For course lists, is it a course? etc.
It's a course
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -800,6 +801,16 @@
...
 
 
+### Jenkins
+
+* [FREE Advanced Jenkins in K8s (Docker in Docker)](https://www.udemy.com/course/free-advanced-jenkins-in-k8s-docker-in-docker) - Nomad Sashi (Udemy)
+* [Jenkins](https://youtube.com/playlist?list=PLhW3qG5bs-L_ZCOA4zNPSoGbnVQ-rp_dG) - Raghav Pal (YouTube)
+* [Jenkins Tutorial](https://www.mygreatlearning.com/academy/learn-for-free/courses/jenkins-tutorial) (Greate Learning)
+* [Jenkins Tutorial Step by Step](https://www.youtube.com/playlist?list=PL8VbCbavWfeGt_Hyq0h9Pymi9DTGowd5X) - Pramod Dutta (YouTube)
+* [Jenkins Tutorial Video [2022 updated]](https://www.youtube.com/playlist?list=PLEiEAq2VkUUKGrfcoNYRgqam5YBERN8xa) - Matthew (Youtube)
+* [Learning Jenkins (2018)](https://www.linkedin.com/learning/learning-jenkins-14423877?replacementOf=learning-jenkins-2018) (Linkedin)
```
Comment (shrn01) [inline courses/free-courses-en.md:811]: Looks like this course is not free

Comment (Manish-Syal123) [reply courses/free-courses-en.md:811]: * [Jenkins Tutorial](https://www.mygreatlearning.com/academy/learn-for-free/courses/jenkins-tutorial) (Greate Learning)
 --->I checked this course and it is free.
ss1

* [Jenkins Tutorial Step by Step](https://www.youtube.com/playlist?list=PL8VbCbavWfeGt_Hyq0h9Pymi9DTGowd5X) - Pramod Dutta (YouTube)
  ---> As this one is from youtube and it is free.
ss2

* [Jenkins Tutorial Video [2022 updated]](https://www.youtube.com/playlist?list=PLEiEAq2VkUUKGrfcoNYRgqam5YBERN8xa) - Matthew (Youtube)
  ---> As this one is also from youtube and it is free.
ss3

Comment (Manish-Syal123) [reply courses/free-courses-en.md:811]: And the last course of linkedin so sorry about that it is only free for a month.
So i am going the replace it with another new course.

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -800,6 +801,16 @@
 * [Vuex for Everyone](https://vueschool.io/courses/vuex-for-everyone)
 
 
+### Jenkins
+
+* [FREE Advanced Jenkins in K8s (Docker in Docker)](https://www.udemy.com/course/free-advanced-jenkins-in-k8s-docker-in-docker) - Nomad Sashi (Udemy)
+* [Jenkins](https://youtube.com/playlist?list=PLhW3qG5bs-L_ZCOA4zNPSoGbnVQ-rp_dG) - Raghav Pal (YouTube)
```
Comment (shrn01) [inline courses/free-courses-en.md:807]:
```suggestion
* [Jenkins](https://youtube.com/playlist?list=PLhW3qG5bs-L_ZCOA4zNPSoGbnVQ-rp_dG) - Raghav Pal
```

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -800,6 +801,16 @@
...
+* [Jenkins for Beginners](https://www.udemy.com/course/jenkins-for-beginners-n) - Musab Zayadneh (Udemy)
+* [Jenkins Tutorial](https://www.mygreatlearning.com/academy/learn-for-free/courses/jenkins-tutorial) (Greate Learning)
```
Comment (shrn01) [inline courses/free-courses-en.md:809]:
```suggestion
* [Jenkins Tutorial](https://www.mygreatlearning.com/academy/learn-for-free/courses/jenkins-tutorial) - My Great Learning
```

File: courses/free-courses-en.md [hunk 4]
```diff
@@ -800,6 +801,16 @@
...
+* [Jenkins Tutorial Step by Step](https://www.youtube.com/playlist?list=PL8VbCbavWfeGt_Hyq0h9Pymi9DTGowd5X) - Pramod Dutta (YouTube)
```
Comment (shrn01) [inline courses/free-courses-en.md:810]:
```suggestion
* [Jenkins Tutorial Step by Step](https://www.youtube.com/playlist?list=PL8VbCbavWfeGt_Hyq0h9Pymi9DTGowd5X) - Pramod Dutta
```

File: courses/free-courses-en.md [hunk 5]
```diff
@@ -800,6 +801,16 @@
...
+* [Jenkins Tutorial Video [2022 updated]](https://www.youtube.com/playlist?list=PLEiEAq2VkUUKGrfcoNYRgqam5YBERN8xa) - Matthew (Youtube)
```
Comment (shrn01) [inline courses/free-courses-en.md:811]:
```suggestion
* [Jenkins Tutorial Video [2022 updated]](https://www.youtube.com/playlist?list=PLEiEAq2VkUUKGrfcoNYRgqam5YBERN8xa) - Matthew
```

**Review by shrn01 — CHANGES_REQUESTED** [review]

**Comment by Manish-Syal123** [comment]
changes have been made . Please have a look

File: courses/free-courses-en.md [hunk 6]
```diff
@@ -804,11 +804,11 @@
...
 
 * [FREE Advanced Jenkins in K8s (Docker in Docker)](https://www.udemy.com/course/free-advanced-jenkins-in-k8s-docker-in-docker) - Nomad Sashi (Udemy)
-* [Jenkins](https://youtube.com/playlist?list=PLhW3qG5bs-L_ZCOA4zNPSoGbnVQ-rp_dG) - Raghav Pal (YouTube)
+* [Jenkins](https://youtube.com/playlist?list=PLhW3qG5bs-L_ZCOA4zNPSoGbnVQ-rp_dG) - Raghav Pal
 * [Jenkins for Beginners](https://www.udemy.com/course/jenkins-for-beginners-n) - Musab Zayadneh (Udemy)
 * [Jenkins Tutorial](https://www.mygreatlearning.com/academy/learn-for-free/courses/jenkins-tutorial) (Greate Learning)
-* [Jenkins Tutorial Step by Step](https://www.youtube.com/playlist?list=PL8VbCbavWfeGt_Hyq0h9Pymi9DTGowd5X) - Pramod Dutta (YouTube)
-* [Jenkins Tutorial Video [2022 updated]](https://www.youtube.com/playlist?list=PLEiEAq2VkUUKGrfcoNYRgqam5YBERN8xa) - Matthew (Youtube)
+* [Jenkins Tutorial Step by Step](https://www.youtube.com/playlist?list=PL8VbCbavWfeGt_Hyq0h9Pymi9DTGowd5X) - Pramod Dutta
+* [Jenkins Tutorial Video [2022 updated]](https://www.youtube.com/playlist?list=PLEiEAq2VkUUKGrfcoNYRgqam5YBERN8xa) - Matthew
```
Comment (shrn01) [inline courses/free-courses-en.md:811]:
```suggestion
* [Jenkins Tutorial Video [2022 updated]](https://www.youtube.com/playlist?list=PLEiEAq2VkUUKGrfcoNYRgqam5YBERN8xa) - Simplilearn
```

File: courses/free-courses-en.md [hunk 7]
```diff
@@ -804,11 +804,11 @@
 ### Jenkins
 
 * [FREE Advanced Jenkins in K8s (Docker in Docker)](https://www.udemy.com/course/free-advanced-jenkins-in-k8s-docker-in-docker) - Nomad Sashi (Udemy)
-* [Jenkins](https://youtube.com/playlist?list=PLhW3qG5bs-L_ZCOA4zNPSoGbnVQ-rp_dG) - Raghav Pal (YouTube)
+* [Jenkins](https://youtube.com/playlist?list=PLhW3qG5bs-L_ZCOA4zNPSoGbnVQ-rp_dG) - Raghav Pal
 * [Jenkins for Beginners](https://www.udemy.com/course/jenkins-for-beginners-n) - Musab Zayadneh (Udemy)
 * [Jenkins Tutorial](https://www.mygreatlearning.com/academy/learn-for-free/courses/jenkins-tutorial) (Greate Learning)
-* [Jenkins Tutorial Step by Step](https://www.youtube.com/playlist?list=PL8VbCbavWfeGt_Hyq0h9Pymi9DTGowd5X) - Pramod Dutta (YouTube)
-* [Jenkins Tutorial Video [2022 updated]](https://www.youtube.com/playlist?list=PLEiEAq2VkUUKGrfcoNYRgqam5YBERN8xa) - Matthew (Youtube)
+* [Jenkins Tutorial Step by Step](https://www.youtube.com/playlist?list=PL8VbCbavWfeGt_Hyq0h9Pymi9DTGowd5X) - Pramod Dutta
```
Comment (shrn01) [inline courses/free-courses-en.md:810]:
```suggestion
* [Jenkins Tutorial Step by Step](https://www.youtube.com/playlist?list=PL8VbCbavWfeGt_Hyq0h9Pymi9DTGowd5X) - The Testing Academy
```

File: courses/free-courses-en.md [hunk 8]
```diff
@@ -804,11 +804,11 @@
 ### Jenkins
 
 * [FREE Advanced Jenkins in K8s (Docker in Docker)](https://www.udemy.com/course/free-advanced-jenkins-in-k8s-docker-in-docker) - Nomad Sashi (Udemy)
-* [Jenkins](https://youtube.com/playlist?list=PLhW3qG5bs-L_ZCOA4zNPSoGbnVQ-rp_dG) - Raghav Pal (YouTube)
+* [Jenkins](https://youtube.com/playlist?list=PLhW3qG5bs-L_ZCOA4zNPSoGbnVQ-rp_dG) - Raghav Pal
```
Comment (shrn01) [inline courses/free-courses-en.md:807]:
```suggestion
* [Jenkins](https://youtube.com/playlist?list=PLhW3qG5bs-L_ZCOA4zNPSoGbnVQ-rp_dG) - Automation Step by Step
```

**Pull request closed without merging**